lib/daru/dataframe.rb

```diff
@@ -545,6 +545,17 @@ def add_vector n, vector
       self[n] = vector
     end
 
+    def insert_vector n, name, source
```

Comment:
- Why it is always necessary to provide name of the index ? 

- Please make the argument names clearly like: use `nthColumn` instead of `n`